### PR TITLE
remove True/False arg in UOps.RANGE

### PR DESCRIPTION
**Pull request by spectralsouls**

### CODE REVIEW

**Comment by spectralsouls** [comment]
will fix the failling tests

File: tinygrad/codegen/linearize.py [hunk 1]
```diff
@@ -37,13 +37,13 @@ def get_recursive_children(x:UOp, end:UOps, include_self=False) -> Set[UOp]:
...
-    if u.op is UOps.RANGE and u.arg[1]:
-      priority += u.arg[0]
+    if u.op is UOps.RANGE and u.arg:
+      priority += u.arg
       for p in range_phi[u]:
         priority += 10000*len([r for r in range_srcs[p] if not any(i in range_phi[u] for i in range_phi[r])])
     # prefer uops that are loop children
     else:
-      priority -= sum([(l.arg[0]+1) + 1000*l.arg[1] for l,ss in scope_children.items() if l.op is UOps.RANGE and u in ss])
+      priority -= sum([(2 if l.arg else 1) + 1000*l.arg for l,ss in scope_children.items() if l.op is UOps.RANGE and u in ss])
```
Comment (geohot) [inline tinygrad/codegen/linearize.py:46]: Not right!

**Comment by geohot** [comment]
Please fix tests and make process replay pass if possible

**Comment by geohot** [comment]
This misses the point! The point is to remove the need for the True/False, not encode it in a different way

**Pull request closed without merging**